ding/framework/tests/test_event_loop.py

```diff
@@ -0,0 +1,52 @@
+from time import sleep
+import pytest
+from ding.framework import EventLoop
+from threading import Lock
+
+
+@pytest.mark.unittest
+def test_event_loop():
+    loop = EventLoop.get_event_loop("test")
+    try:
+        counter = 0
+        lock = Lock()
+
+        def callback(n, lock):
+            nonlocal counter
+            with lock:
+                counter += n
+
+        # Test on
+        loop.on("count", callback)
+
+        for i in range(5):
+            loop.emit("count", i, lock)
+        sleep(0.1)
+        assert counter == 10
```

Comment:
call emit when `on` at least twice, which can be comparison with `once` case

Reply:
I don't think it's necessary, once has been tested below.